models/articles.models.js

```diff
@@ -13,7 +13,24 @@ const fetchArticleById = (id) => {
     });
 };
 
-const fetchArticles = (sort_by = "created_at", order = "desc") => {
+const checkTopics = (topic) => {
+  return db
+    .query(
+      `SELECT * FROM topics
+      WHERE slug = $1`,
+      [topic]
+    )
+    .then((result) => {
+      console.log(result.rows);
+      if (result.rows.length > 0 || topic === undefined) {
+        return result.rows.slug;
+      } else if (result.rows.length === 0) {
+        return Promise.reject({ status: 400, message: "Bad Request" });
+      }
+    });
+};
```

Comment:
You've made checkTopics, but then haven't used it for this ticket, as it doesn't appear in fetchArticles. I'm not _sure_ why this has been created here and then not used, as your fetchArticles works as it should :)